openretina/hoefling_2024/nnfabrik_model_loading.py

```diff
@@ -250,22 +260,26 @@ def load_ensemble_retina_model_from_directory(directory_path: str, device: str =
 
         state_dict = torch.load(state_dir_path)
         with open(model_config_path, "r") as f:
-            config = yaml.safe_load(f)
-
+            config = yaml.load(f, SafeLoaderWithTuple)
         with open(data_info_path, "rb") as fb:
             data_info = pickle.load(fb)
         data_info_list.append(data_info)
 
         model_fn = config["model_fn"]
+        [repo, _, _, model_type] = model_fn.split('.')
```

Comment:
parenthesis are not required.
```suggestion
        repo, _, _, model_type = model_fn.split('.')
```